digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/TierekisteriCarryingCapacityAssetClient.scala

```diff
@@ -0,0 +1,36 @@
+package fi.liikennevirasto.digiroad2.client.tierekisteri
+
+import java.util.Date
+import fi.liikennevirasto.digiroad2.util.Track
+import org.apache.http.impl.client.CloseableHttpClient
+
+case class TierekisteriCarryingCapacityData(roadNumber: Long, startRoadPartNumber: Long, endRoadPartNumber: Long,
+                                        track: Track, startAddressMValue: Long, endAddressMValue: Long, springCapacity: Option[String],
+                                        factorValue: Int, measurementDate: Option[Date]) extends TierekisteriAssetData
+
+class TierekisteriCarryingCapacityAssetClient(trEndPoint: String, trEnable: Boolean, httpClient: CloseableHttpClient) extends TierekisteriAssetDataClient{
+  override def tierekisteriRestApiEndPoint: String = trEndPoint
+  override def tierekisteriEnabled: Boolean = trEnable
+  override def client: CloseableHttpClient = httpClient
+  type TierekisteriType = TierekisteriCarryingCapacityData
+
+  override val trAssetType = "tl210"
+  private val trSpringCarryingCapacity = "KEVKANT"
+  private val trFrostHeavingFactor = "K2"
+  private val trDateOfMeasurement = "ALKUPVM"
+
+  override def mapFields(data: Map[String, Any]): Option[TierekisteriCarryingCapacityData] = {
+    val springCapacity = getFieldValue(data, trSpringCarryingCapacity)
+    val factorValue = convertToInt(getFieldValue(data, trFrostHeavingFactor)).orElse(Some(TRFrostHeavingFactorType.Unknown.value)).get
```

Comment:
is it possible to change **convertToInt(getFieldValue(data, trFrostHeavingFactor)).orElse(Some(TRFrostHeavingFactorType.Unknown.value)).get**  to **convertToInt(getFieldValue(data, trFrostHeavingFactor)).getorElse(TRFrostHeavingFactorType.Unknown.value)**?